src/NSubstitute.Analyzers.VisualBasic/DiagnosticAnalyzers/SubstitutionNodeFinder.cs

```diff
@@ -17,6 +18,7 @@ private SubstitutionNodeFinder()
     {
     }
 
+    // TODO replace with SyntaxVisitor or OperationVisitor
```

Comment:
This will be handled in separate PR. Most of the work to move to operation API will be done as part of https://github.com/nsubstitute/NSubstitute.Analyzers/issues/153